src/pages/MovieList/styled.jsx

```diff
@@ -0,0 +1,233 @@
+import styled from "styled-components";
+import { ReactComponent as star } from "../../assets/star.svg";
+
+export const Container = styled.div`
+  margin: 0 auto;
+  display: flex;
+  margin-top: 50px;
+  flex-direction: column;
+  justify-content: center;
+`;
+
+export const Content = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
+  align-self: center;
+  grid-gap: 10px;
```

Comment:
grid-gap nie będzie działał jeśli display jest ustawiony na flex, trzeba to zmienić :)